Utility.cpp

```diff
@@ -27,7 +30,7 @@ std::vector<std::unique_ptr<Item>> makeDefensiveItems(int num)
         items.push_back( std::unique_ptr<DefensiveItem>(new DefensiveItem()) );
     }
     
-    std::cout << "made " << items.size() << " defensive items" << std::endl;
+    // std::cout << "made " << items.size() << " defensive items" << std::endl;
```

Comment:
These lines did not need to be removed. 
It is important to learn when to leave code alone and when to modify code.
You never know who is depending on code that you are changing.
-[ ] revert